OUDS/Core/OUDS/Sources/OUDSTheme/OUDSTheme+SemanticTokens/OUDSTheme+SpaceMultipleSemanticTokens.swift

```diff
@@ -0,0 +1,38 @@
+//
+// Software Name: OUDS iOS
+// SPDX-FileCopyrightText: Copyright (c) Orange SA
+// SPDX-License-Identifier: MIT
+//
+// This software is distributed under the MIT license,
+// the text of which is available at https://opensource.org/license/MIT/
+// or see the "LICENSE" file for more details.
+//
+// Authors: See CONTRIBUTORS.txt
+// Software description: A SwiftUI components library with code examples for Orange Unified Design System
+//
+
+import Foundation
+import OUDSTokensRaw
+import OUDSTokensSemantic
+
+// [File not generated by the tokenator]
+// WARNING: Not synchronized anymore with the Figjam / Figma by developers team
+// Create an issue for update https://github.com/Orange-OpenSource/ouds-ios/issues/new?template=token_update.yml
+
+/// Defines wrapper objects for space semantic tokens (i.e. in the end ``DimensionRawToken`).
+/// These values can be overriden inside ``OUDSTheme`` subclasses (in extensions or not, in the same module or not) thanks to the `@objc open` combination.
+extension OUDSTheme: SpaceMultipleSemanticTokens {
+
+    // MARK: Semantic token - Spacing - Layout fluid
+
+    @objc open var spaceScaledNone: MultipleSpaceTokens { MultipleSpaceTokens(compact: spaceScaledNoneMobile, regular: spaceScaledNoneTablet) }
```

Comment:
For all Multiple token should we prefer MultipleSementic ?
Example MultipleSpaceTokens shoould be something like SpaceMultipleSementicToken